spec/lib/vanagon/project/dsl_spec.rb

```diff
@@ -231,6 +232,35 @@
     end
   end
 
+  describe "#custom_build_var" do
+    let(:project_block) {
+"project 'test-fixture' do |proj|
+  proj.custom_build_var \"TEST_VAR='foo'\"
+end"
```

Comment:
I was mostly just following how we were doing this elsewhere, but *shrug*. I can change it if you want.